nixos/modules/installer/tools/nixos-option/default.nix

```diff
@@ -0,0 +1,10 @@
+{stdenv, boost, cmake, pkgconfig, nix, ... }:
+stdenv.mkDerivation rec {
+  name = "nixos-option";
+  src = ./.;
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ boost nix ];
+  meta = {
+    license = stdenv.lib.licenses.lgpl2Plus;
```

Comment:
maintainer would probably be good here too

Reply:
Done